Gems/Atom/Feature/Common/Assets/Materials/Pipelines/LowEndPipeline/ForwardPass.azsli

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+// These setting used to be in ShaderQualityOptions.azsli as a basic/limited solution for multiple pipelines.
+// TODO, instead of UNIFIED_FORWARD_OUTPUT, fork ForwardPassOutput.azsli
```

Comment:
Suggest using a searchable string like `TODO(MaterialPipeline)` to make things easy to find later

Reply:
Sure, I can do that.